docs/administration.platforms.md

```diff
@@ -41,3 +74,28 @@ If you want to remove quotas from a platform just click the "-" button on the ri
 > Currently only OpenShift is supported for Quota Management
 
 ![Manage Platform Quota Definitions](assets/tenants/platform-quota-definitions.png)
+
+### Platform Deactivation
+
+Via the "Danger Zone" tab you can deactivate or reactivate your meshPlatform.
+
+Deactivation will make the meshPlatform not available anymore to new meshTenants, existing meshTenants won't be accessible via meshPanel anymore and no more replication will be executed. Metering instead will still be executed for non-deleted meshTenants of this meshPlatform for now. This will change in future. Direct access in the cloud platform itself to the platform tenants is still possible as meshStack does not delete or do any updates on platform tenants when a meshPlatform is deactivated.
+
+A reactivation of meshPlatforms is also possible. This will allow reaccessing and replicating non-deleted meshTenants of this meshPlatform again and make the meshPlatform available again for new meshTenants.
+
+## Platform Notifications
+
+Administrators can inform users about platform specific events (updates, new features, service disruptions, etc.) by creating platform
+notifications:
+
+- **Platforms**: Select the platforms relevant to your notification so they can be shown on the affected platform's dashboards. Users
+  can also [subscribe](meshcloud.profile.md#profile) to notifications by platform and [subscribed users](./meshcloud.profile.md#platform-notification-subscriptions) will receive new notifications via email.
+- **Message**: The message that will be shown to all users. You can enter valid HTML in this field (except for the `<p>` tag). This is useful for e.g. supplying links to support pages.
+- **Severity**: The type of message (info, warning, critical).
+  - Info: for new features, upcoming services updates and general information that is not expected to disrupt platform operation
+  - Warning: best used to inform about partial platform outages or problems that may occur for some users
+  - Critical: appropriate in case of complete platform failures or impending platform downtimes
+- **Show From**: The earliest time the message will be shown. You can use this to create messages that will only be shown on at a later date.
+- **Show Until**: After this time the message will no longer be displayed.
```

Comment:
For these two values there is also behavior when leaving them empty right? Maybe describe that too.